pkg/lib/operation/template/local/create/operation.go

```diff
@@ -27,13 +31,59 @@ type Options struct {
 	Id          string
 	Name        string
 	Description string
+	Branch      model.BranchKey
 	Configs     []ConfigDef
 }
 
+func (o *Options) ObjectsFilter() model.ObjectsFilter {
+	var keys []model.Key
+
+	// Branch
+	keys = append(keys, o.Branch)
+
+	// Configs and rows
+	for _, config := range o.Configs {
+		keys = append(keys, config.Key)
+		for _, row := range config.Rows {
+			keys = append(keys, row.Key)
+		}
+	}
+
+	filter := model.NoFilter()
+	filter.SetAllowedKeys(keys)
+	return filter
+}
+
+func (o *Options) Replacement() replacekeys.Keys {
```

Comment:
Jen kosmetická drobnost, možná by to bylo lepší taky v množným čísle (`Replacements`)?

Reply:
:+1: